config/typescript/tsconfig.prod.json

```
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "outDir": "../../dist",
+    "outDir": "../../dist/vvd-components",
```

why not plain components?
```suggestion
    "outDir": "../../dist/components",
```